docs/CONTRIBUTING-id.md

```diff
@@ -16,23 +16,23 @@ Dengan berkontribusi, Anda setuju untuk menghormati [Kode Etik](CODE_OF_CONDUCT-
 <!----><a id="in-a-nutshell"></a>
 ## Versi pendek
 
-1. "Tautan untuk mengunduh buku dengan mudah" tidak selalu merujuk pada buku yang *gratis*. Harap hanya memberikan konten yang gratis. Pastikan itu benar-benar gratis. Kami tidak menerima tautan ke halaman yang *meminta* alamat email yang valid untuk mengunduh buku, tetapi kami menyambut daftar situs yang meminta alamat email.
+1. "Tautan untuk mengunduh buku dengan mudah" tidak selalu merujuk pada buku yang *gratis*. Harap hanya mendaftarkan tautan yang gratis. Pastikan itu benar-benar gratis. Kami tidak menerima tautan ke halaman yang *meminta* alamat email yang valid untuk mengunduh buku, tetapi kami menyambut daftar situs yang meminta alamat email.
```

Comment:
that's too ambiguous

Reply:
Yeah, agree with that, will try to paraphrase it